shimmingtoolbox/cli/b1shim.py

```diff
@@ -67,15 +72,46 @@ def b1shim_cli(fname_b1_map, fname_mask, algorithm, target, fname_vop, sed, path
         vop = None
 
     shim_weights = b1shim(b1_map, mask=mask_resampled, algorithm=algorithm, target=target,
-                          q_matrix=vop, sed=sed, path_output=path_output)
+                          q_matrix=vop, sed=sed)
+
+    # Indicate output path to the user
+    print(f"\nB1+ shimming results located in: {path_output}\n")
 
     # Write to a text file
-    fname_output = os.path.join(path_output, 'RF_shim_weights.txt')
+    fname_output = os.path.join(path_output, 'b1_shim_weights.txt')
     file_rf_shim_weights = open(fname_output, 'w')
     file_rf_shim_weights.write(f'Channel\tmag\tphase (\u00b0)\n')
     for i_channel in range(len(shim_weights)):
         file_rf_shim_weights.write(f'Tx{i_channel + 1}\t{np.abs(shim_weights[i_channel]):.3f}\t'
                                    f'{np.rad2deg(np.angle(shim_weights[i_channel])):.3f}\n')
     file_rf_shim_weights.close()
 
+    # Plot B1+ shimming results
+    b1_shimmed = montage(combine_maps(b1_map, shim_weights))  # RF-shimming result
+    if mask_resampled is not None:
+        b1_shimmed_masked = b1_shimmed*montage(mask_resampled)
+    else:
+        b1_shimmed_masked = b1_shimmed*montage(threshold(b1_map.sum(axis=-1), thr=0))
+
+    b1_shimmed_masked[b1_shimmed_masked == 0] = np.nan  # Replace 0 values by nans for image transparency
+    vmax = np.percentile(b1_shimmed, 99)  # Reduce high values influence on display
+    vmax = 5*np.ceil(vmax/5)  # Ceil max range value to next multiple of 5 for good colorbar display
+
+    plt.figure()
+    plt.imshow(b1_shimmed, vmax=vmax, cmap='gray')  # Display background in gray
+    plt.imshow(b1_shimmed_masked, vmin=0, vmax=vmax, cmap="jet")  # Overlay colored shimming ROI
```

Comment:
@po09i that might be of interest to you too as we discussed using jet for your plots

Reply:
Other possibility: use a diverging cmap centered on the mean B1 value: 
![image](https://user-images.githubusercontent.com/54723104/153634505-08ed836f-d77c-415c-8da7-544e2b426f68.png)

Pros: slight inhomogeneities are twice easier to visualize because of the 2 colormaps 
Cons: Not the most intuitive way to visualize data (at least in my opinion)

Reply:
how about Viridis?

![image](https://user-images.githubusercontent.com/2482071/153639182-c80e5e76-6577-4867-9252-26a84092d774.png)

Reply:
or hot?

![image](https://user-images.githubusercontent.com/2482071/153639287-e0d74108-877f-4ef0-bb78-5d13433699c2.png)

Reply:
## 'viridis'

![image](https://user-images.githubusercontent.com/54723104/153639499-618a87a8-080f-4898-bba3-8d12ce9649dc.png)

Reply:
"hot" is not perfectly sequential: "Others (afmhot, copper, gist_heat, and hot) have kinks in the  functions" 

So I think the problem that we have with 'jet' would also apply here

Reply:
Regarding the "Turbo" colormap, it is not natively included in Matplotlib...

Reply:
I decided to stick to the default Viridis in the end after watching [this](https://www.youtube.com/watch?v=xAoljeRJ3lU).